api/swagger.yml

```diff
@@ -3516,3 +3516,35 @@ paths:
                 $ref: "#/components/schemas/StorageConfig"
         401:
           $ref: "#/components/responses/Unauthorized"
+  /templates:
+    get:
+      tags:
+        - templates
+      operationId: expandTemplate
+      description: fetch and expand template
+      parameters:
+        - in: query
+          name: template_location
+          required: true
+          schema:
+            type: string
+            description: URL of the template; must be relative (to a URL configured on the server).
+        - in: query
+          name: params
+          schema:
+            type: object
+            additionalProperties:
+              type: string
+      responses:
+        200:
+          description: expanded template
+          content:
+            "*/*":
+              schema:
+                format: binary
+        401:
+          $ref: "#/components/responses/Unauthorized"
```

Comment:
What about 403 Forbidden?
The user might be authenticated but not authorized to fetch the template or use the requested function

Reply:
For some reason we *never* specify 403 in this file.  @nopcoder do you know why?  In any case, I'm not blocking on this.